lte/gateway/fabfile.py

```diff
@@ -409,14 +409,6 @@ def bazel_integ_test_post_build(
     else:
         ansible_setup(test_host, "test", "magma_test.yml")
 
-    execute(_make_integ_tests)
```

Comment:
FYI: These steps are now done in the `vagrant ssh` call in the workflow. The Make workflow is unchanged as that is in a different fabric function.